testdata/integration_tests/SCHEDULE/SCHEDULE_COMPDAT1

```diff
@@ -1,3 +1,19 @@
+DIMENS
```

Comment:
Oh yeah. I'll update that, essentially I just put in something massive to make sure I didn't get out-of-bounds errors.